docs/source/entities-advanced.mdx

```diff
@@ -0,0 +1,604 @@
+    * When this deployment completes, the gateway begins resolving `Bill` fields in the Billing subgraph _instead of_ the Payments subgraph.
+
+7. Deploy an updated version of your Payments subgraph without the `Bill` entity.
+    * At this point it's safe to remove this definition, because the gateway is never resolving `Bill` with the Payments subgraph.
+
+We're done! `Bill` is now resolved in a new subgraph, and it was resolvable during each step of the migration process.
+
+</ExpansionPanel>
+
+<ExpansionPanel title="With managed federation">
+
+> During the Federation 2 preview, this process uses the `@shareable` directive to make sure your supergraph schema composes during each step of the migration process. The full release of Federation 2 will include a different mechanism _specifically_ designed for migrating fields.
+
+1. In the Payments subgraph's schema, add the `@shareable` directive to the `Bill` entity's definition:
+
+    <CodeColumns>
+
+    ```graphql{2}
+    # Payments subgraph
+    type Bill @key(fields: "id") @shareable {
+      id: ID!
+      amount: Int!
+    }
+
+    type Payment {
+      # ...
+    }
+    ```
+
+    </CodeColumns>
+
+    This allows other subgraphs to resolve all `Bill` fields defined in Payments.
+
+2. Publish the updated Payments subgraph schema to Apollo.
+
+    * When the gateway fetches its new supergraph schema, its behavior doesn't change. Even though the `Bill` entity is now `@shareable`, the Payments subgraph is still the only subgraph that defines it.
+
+3. In the Billing subgraph's schema, define the `Bill` entity exactly as it's now defined in the Payments subgraph:
+
+    <CodeColumns>
+
+    ```graphql
+    # Payments subgraph
+    type Bill @key(fields: "id") @shareable {
+      id: ID!
+      amount: Int!
+    }
+
+    type Payment {
+      # ...
+    }
+    ```
+
+    ```graphql{2-5}
+    # Billing subgraph
+    type Bill @key(fields: "id") @shareable {
+      id: ID!
+      amount: Int!
+    }
+    ```
+
+    </CodeColumns>
+
+4. Also in the Billing subgraph, define resolvers for each field of `Bill` that's currently resolved in the Payments subgraph. These resolvers should behave identically to the resolvers in the Payments subgraph.
+
+5. Deploy the updated Billing subgraph to your environment.
+
+    * This change is currently invisible to your gateway, which is not yet aware that the Billing subgraph defines the `Bill` entity.
+
+6. Publish the updated Billing subgraph schema to Apollo.
+
+    * When the gateway fetches its new supergraph schema, it detects that both the Payments _and_ Billing subgraphs can resolve fields of the `Bill` entity.
+
+7. In the Payments subgraph, _remove_ the `Bill` entity and its associated resolvers:
+
+    <CodeColumns>
+
+    ```graphql
+    # Payments subgraph
+    type Payment {
+      # ...
+    }
+    ```
+
+    ```graphql
+    # Billing subgraph
+    type Bill @key(fields: "id") {
```

Comment:
```suggestion
    type Bill @key(fields: "id") @shareable {
```

@StephenBarlow — I think this `Bill` entity in the Billing subgraph should still have the `@shareable` directive at this point? (you tell them to remove `@shareable` in step 10 below).

Reply:
Great catch thank you!!